apps/actors/lib/actors/actor/entity.ex

```diff
@@ -230,6 +230,41 @@ defmodule Actors.Actor.Entity do
       current_context: Context.new(state: current_state)
     )
     |> invoke_host(state)
+    |> case do
+      {:ok, response, state} -> {:reply, {:ok, response}, state}
+      {:error, :no_content, state} -> {:reply, {:error, :no_content}, state}
+      {:error, reason, state} -> {:reply, {:error, reason}, state, :hibernate}
+    end
+  end
+
+  @impl true
+  def handle_cast(
+        {:invocation_request,
+         %InvocationRequest{
+           actor: %Actor{name: name} = _actor,
+           command_name: command,
+           value: payload
+         } = _invocation},
+        %EntityState{
+          system: actor_system,
+          actor: %Actor{state: actor_state}
+        } = state
+      ) do
+    current_state = Map.get(actor_state || %{}, :state)
+
+    ActorInvocation.new(
+      actor_name: name,
+      actor_system: actor_system,
+      command_name: command,
+      value: payload,
+      current_context: Context.new(state: current_state)
+    )
+    |> invoke_host(state)
+    |> case do
+      {:ok, _whatever, state} -> {:noreply, state}
+      {:error, :no_content, state} -> {:noreply, state}
```

Comment:
I think it's good to put him to sleep in all cases of error (remembering that he will only sleep anyway if his run queue is empty)